src/AspNet.Security.OAuth.Discord/DiscordAuthenticationHandler.cs

```diff
@@ -29,6 +31,21 @@ public class DiscordAuthenticationHandler : OAuthHandler<DiscordAuthenticationOp
         {
         }
 
+        /// <inheritdoc />
+        protected override string BuildChallengeUrl(
+            [NotNull] AuthenticationProperties properties,
+            [NotNull] string redirectUri)
+        {
+            string challengeUrl = base.BuildChallengeUrl(properties, redirectUri);
+
+            if (!string.IsNullOrWhiteSpace(Options.DiscordAuthenticationPrompt))
```

Comment:
In general, we tend to use `string.IsNullOrEmpty`, which is consistent with what ASP.NET and .NET in general do.

Reply:
Okay, I will make that change.